index.html

```diff
@@ -103,21 +103,23 @@
 
 ## Principles
 
-<dl>
-<dt><dfn id="oneweb" data-export="">
-  There is one web
-</dfn></dt>
-<dd>
+<section>
+  <h3 id="oneweb" data-export="">
+There is one web
+  </h3>
+  <p>
```

Comment:
You should be able to omit the `<p>`s since you have markdown enabled. Just be sure to have a blank line around paragraphs.